From 903779df9a65ac04814afb1530423914bdfe305c Mon Sep 17 00:00:00 2001
From: Lars Ellenberg <lars.ellenberg@linbit.com>
Date: Sun, 6 Oct 2024 01:14:19 +0200
Subject: [PATCH 31/32] build: fix typo in Makefile.spatch

The variable name should be cached_compat_h, compat_h does not exist.
Noticed when built with `make -j`:
Because of the empty prerequisites, sometimes make would try to build
kernelrelease.txt before the directory was created.
---
 drbd/Makefile.spatch | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drbd/Makefile.spatch b/drbd/Makefile.spatch
index bbcf10dc0603..a5cc774c18b5 100644
--- a/drbd/Makefile.spatch
+++ b/drbd/Makefile.spatch
@@ -42,7 +42,7 @@ cached_compat_patch := $(cache_dir)/compat.patch
 
 # Remember which kernel releases this supposedly works for.
 # If KERNELRELEASE is not known, will add an empty line (at most once).
-$(cache_dir)/kernelrelease.txt: $(compat_h) FORCE
+$(cache_dir)/kernelrelease.txt: $(cached_compat_h) FORCE
 	$(Q)set -e;						\
 	grep -sqxFe "$(KERNELRELEASE)" $@ < /dev/null ||	\
 	echo $(KERNELRELEASE) >> $@;
-- 
2.35.3

